selfdrive/updated.py

```diff
@@ -367,10 +380,7 @@ def main() -> None:
 
   first_run = True
   last_fetch_time = 0.0
-  update_failed_count = 0
-
-  # Set initial params for offroad alerts
-  set_params(False, 0, None)
```

Comment:
This would set `LastUpdateTime` to the current time when starting updated. Was that intentional?